network/wifi/wifi.c

```diff
@@ -166,7 +160,7 @@ int32_t wifi_init(struct wifi_desc **desc, struct wifi_init_param *param)
 
 	wifi_init_intarface(ldesc);
 
-	result = at_run_cmd(ldesc->at, AT_RESET, AT_EXECUTE_OP, NULL);
+	//result = at_run_cmd(ldesc->at, AT_RESET, AT_EXECUTE_OP, NULL);
```

Comment:
Is the reset no longer needed?

Reply:
Was used just for faster debugging. I shouldn't have commited this change